libs/platform/experience/entity-text/entity-text.schema.ts

```diff
@@ -0,0 +1,18 @@
+import { ContentComponentSchema } from '@spryker-oryx/experience';
+import { EntityTextComponent } from './entity-text.component';
+import { FormFieldType } from '@spryker-oryx/form';
+import { IconTypes } from '@spryker-oryx/ui/icon';
+
+export const entityTextSchema: ContentComponentSchema<EntityTextComponent> = {
+  name: 'Entity text',
+  group: 'Experience',
+  icon: IconTypes.Edit,
+  options: {
+    entity: {
+      type: FormFieldType.Text,
+    },
+    field: {
+      type: FormFieldType.Text,
+    },
```

Comment:
can we centralise those and spread them in?

Reply:
hmm... not strighforward in a clean way. When we will put it into src, then it will become part of main bundle, and this is part of schema, that we preffer to lazy loadin EB 🤔